sklearn/decomposition/tests/test_nmf.py

```diff
@@ -42,25 +42,28 @@ def test_initialize_nn_output():
 def test_parameter_checking():
     A = np.ones((2, 2))
     name = 'spam'
+    # FIXME : should be removed in 0.26
+    init = 'nndsvda'
```

Comment:
Instead of changing the init everywhere, you can filter the corresponding warning for the whole test:
``@pytest.mark.filterwarnings("ignore:will be changed from 'nndsvd' to 'nndsvda'")``

Reply:
The idea was to make sure that tests will pass with the new initialization, not only to avoid warnings (see [this comment](https://github.com/scikit-learn/scikit-learn/pull/18525#discussion_r501187975)).

Reply:
right, then it's probably better this way, sorry for the noise :)